src/components/gallery/CurrentImg.tsx

```diff
@@ -0,0 +1,168 @@
+import { useEffect, useState } from "react";
+import * as style from "./CurrentImgStyle";
+import { getStorage, ref, deleteObject } from "firebase/storage";
+import { app } from "@/firebase/firebase";
+
+const storage = getStorage(app);
+
+interface CurrentImgProps {
+  curImg: string;
+  imagePaths: string[];
+  setViewImg: React.Dispatch<React.SetStateAction<boolean>>;
+  setCurImg: React.Dispatch<React.SetStateAction<string>>;
+  setImagePaths: React.Dispatch<React.SetStateAction<string[]>>;
+}
+
+export default function CurrentImg({
+  setViewImg,
+  setCurImg,
+  setImagePaths,
+  curImg,
+  imagePaths,
+}: CurrentImgProps) {
+  const [prevBtn, setPrevBtn] = useState(true);
+  const [nextBtn, setNextBtn] = useState(true);
+  const [curIndex, setCurIndex] = useState(0);
+  const [imgScale, setImgScale] = useState(1);
+
+  useEffect(() => {
+    const index = imagePaths.findIndex((imagePath) => imagePath === curImg);
+    setCurIndex(index);
+  }, [curIndex]);
+
+  const nextImg = () => {
+    console.log(curIndex);
+    if (curIndex >= imagePaths.length - 2) {
+      const cur = imagePaths[curIndex + 1];
+      let num = curIndex;
+      num++;
+      setCurIndex(num);
+      setCurImg(cur);
+      setNextBtn(false);
+    } else {
+      setNextBtn(true);
+      let num = curIndex;
+      num++;
+      setCurIndex(num);
+      const cur = imagePaths[curIndex + 1];
+      setCurImg(cur);
+    }
+
+    if (curIndex >= 0) {
+      setPrevBtn(true);
+    }
+  };
+
+  const prevtImg = () => {
+    console.log(curIndex);
+    if (curIndex <= 1) {
+      let num = curIndex;
+      num--;
```

Comment:
```suggestion
      const num = curIndex - 1;
```

요롷게 하면 한 줄에 작성할 수 있어보입니당